### ISSUE TITLE
Buttons not working

### ISSUE THREAD

**Issue by MartenvanWezel** [RichMercer/SmartThings#1]
As far as I can tell you never got round to (or were able to) get the actual buttons of the 5-button remote to work. True? Or if they do work... how?

**Comment by RichMercer** [comment]
Nope, I wasn't able to get this to work. It's been a while since I've looked at it, but from what I understood (and remember) the remote talks to the lights directly, and doesn't communicate as a button with the hub. The only way to get it to work is to somehow get the button to see the hub as a light so it sends the commands to the hub that way. Then the hub could read them and act on them. Not sure if it's possible or not, but that was the problem.

**Comment by MartenvanWezel** [comment]
Hey, of course if you're "done with it" that's cool, but as I understand it IKEA fairly recently-ish (Oct?) has made the button a lot more zigbee compliant. It "should" be more doable at this point.

**Comment by RichMercer** [comment]
I didn't realise that so I'll reopen this and see if I can get some time to look at it some time.

**Comment by MartenvanWezel** [comment]
The entire community would be FOREVER in your debt. (or at least until ...
ooh new shiney..... )
Seriously, the smartthings, ikea community boards are overrunning with
people requesting this, and you seem to be the closest to getting this
done.. There will be a lot of glory involved ;)

**Comment by hkusulja** [comment]
I am not developer, but, this is not working, any ETA on status or fixing this? Or just write in README that this does not work to spare some time for others.., tx

**Comment by RichMercer** [comment]
I updated the README to reflect that this isn't something that can be done currently. It's frustrating but perhaps a future update will allow it to work.

**Comment by hkusulja** [comment]
Can you also just write, the current version number that you have tried on with?
